operator-common/src/test/java/io/strimzi/operator/common/operator/MockCertManager.java

```diff
@@ -156,6 +158,10 @@ public static String clientsCaCertStore() {
         return Base64.getEncoder().encodeToString(CLIENTS_CERT_STORE);
     }
 
+    public static String clientsCaCertStorePassword() {
+        return Base64.getEncoder().encodeToString(CLIENTS_CERT_STORE_PASSWORD.getBytes(Charset.defaultCharset()));
+    }
+
```

Comment:
How is the secret mocked in other tests without this method? Maybe you can use the same instead of adding this?

Reply:
It is hard-coded in the other tests e.g. [1] [2] I figured we could refactor the other occurances in a separate PR as a lot of other reference [3]

[1] https://github.com/strimzi/strimzi-kafka-operator/blob/201540481bdf48c45056e7204ee799eaf3b7b4cf/certificate-manager/src/test/java/io/strimzi/certs/OpenSslCertManagerTest.java#L74
[2] https://github.com/strimzi/strimzi-kafka-operator/blob/76cd8065b22c8cf317d6666b271f306d54ea0bd3/cluster-operator/src/test/java/io/strimzi/operator/cluster/ResourceUtils.java#L211
[3] https://github.com/strimzi/strimzi-kafka-operator/search?q=123456